src/test/msgr/test_async_networkstack.cc

```diff
@@ -41,17 +41,17 @@ class NetworkWorkerTest : public ::testing::TestWithParam<const char*> {
   void SetUp() override {
     cerr << __func__ << " start set up " << GetParam() << std::endl;
     if (strncmp(GetParam(), "dpdk", 4)) {
-      g_ceph_context->_conf->set_val("ms_type", "async+posix", false, false);
+      g_ceph_context->_conf->set_val("ms_type", "async+posix", false);
       addr = "127.0.0.1:15000";
       port_addr = "127.0.0.1:15001";
     } else {
-      g_ceph_context->_conf->set_val("ms_type", "async+dpdk", false, false);
-      g_ceph_context->_conf->set_val("ms_dpdk_debug_allow_loopback", "true", false, false);
```

Comment:
OPTION(ms_dpdk_debug_allow_loopback, OPT_BOOL, false)

It's OPT_BOOL, no need to pass `safe=false`